src/event/player/PlayerViewDistanceChangeEvent.php

```diff
@@ -29,20 +29,27 @@
  * Called when a player requests a different viewing distance than the current one.
  */
 class PlayerViewDistanceChangeEvent extends PlayerEvent{
-	protected int $newRadius;
-	protected int $oldRadius;
+	protected int $newDistance;
+	protected int $oldDistance;
 
-	public function __construct(Player $player, int $oldRadius, int $newRadius){
+	public function __construct(Player $player, int $oldDistance, int $newDistance){
 		$this->player = $player;
-		$this->oldRadius = $oldRadius;
-		$this->newRadius = $newRadius;
+		$this->oldDistance = $oldDistance;
+		$this->newDistance = $newDistance;
 	}
 
-	public function getNewRadius() : int{
-		return $this->newRadius;
+	/**
+	 * Returns an int corresponding to the number of chunks forming the new view distance
+	 */
+	public function getNewDistance() : int{
+		return $this->newDistance;
 	}
 
-	public function getOldRadius() : int{
-		return $this->oldRadius;
+	/**
+	 * Returns an int corresponding to the number of chunks forming the previous view distance, before change
+	 * If the player connects to the server, it will return -1
```

Comment:
```suggestion
	 * A value of -1 means that the player has just connected and did not have a view distance before this event.
```

Reply:
As this is very clear, I have not been able to express it more clearly.

Reply:
"if the player connects" is unclear. Every player connects to the server ... does that mean it will always return -1? No.